coffee-shop/src/pages/Order/components/PaymentSummary/PaymentSummary.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import styles from "./PaymentSummary.module.css";
+
+const PaymentSummary = ({ price, deliveryFee }) => (
+  <div className={styles.paymentSummaryContainer}>
+    <div className={styles.paymentSummaryTitle}>Payment Summary</div>
+    <div className={styles.paymentSummaryItem}>
+      <span>Price</span>
+      <span>
+        <b>${price.toFixed(2)}</b>
+      </span>
+    </div>
+    <div className={styles.paymentSummaryItem}>
+      <span>Delivery Fee</span>
+      <span>
+        <span className={styles.originalFee}>
```

Comment:
should be calculation between fees and discount

Reply:
Done